extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -53,11 +53,9 @@ class AmpYoutube extends AMP.BaseElement {
     /** @private {number} */
     this.playerState_ = 0;
 
-    // The video-id is supported only for backward compatibility.
     /** @private @const {string} */
     this.videoid_ = user().assert(
-        (this.element.getAttribute('data-videoid') ||
-        this.element.getAttribute('video-id')),
+        this.element.getAttribute('data-videoid'),
```

Comment:
This matches master in the main repo... some how I picked up changes I did not intend.  Not sure how to force revert.  Apologies for my weak git kung fu.